src/main/java/com/google/dicomwebfuse/dao/FuseDaoHelper.java

```diff
@@ -146,12 +170,22 @@
       instancesList.addAll(resultInstancesList);
     }
     if (instancesList.size() > MAX_INSTANCES_IN_SERIES) {
-      throw new ToManyResultsException("Too large Series. In " + queryBuilder.getSeriesId() +
+      LOGGER.warn("Too large Series. In " + queryBuilder.getSeriesId() +
           " over " + MAX_INSTANCES_IN_SERIES + " Instances!");
     }
     return instancesList;
   }
 
+  public static Instance getInstance(FuseDao fuseDao, CloudConf cloudConf, DicomPath dicomPath) throws DicomFuseException {
```

Comment:
getSingleInstance for consistency with getSingleSeries?

Reply:
done